### PR TITLE
markup in title attribute does not render in tooltip

### PR DESCRIPTION
**Pull request by brianteeman**
PR for #19074

I could have made the "title" attribute a bootstrap tooltip for consistency but after I had done that I realised that the tooltip was identical to the text it was a tooltip for. Thats just crazy so I completely removed the "title" attribute as its superfluous.

### CODE REVIEW

File: administrator/components/com_config/model/field/filters.php [hunk 1]
```diff
@@ -51,16 +51,13 @@ protected function getInput()
...
 		$html[] = '		</th>';
 		$html[] = '		<th>';
-		$html[] = '			<span class="acl-action" title="' . JText::_('JGLOBAL_FILTER_TYPE_LABEL') . '">'
-				. JText::_('JGLOBAL_FILTER_TYPE_LABEL') . '</span>';
+		$html[] = '			<span class="acl-action">' . JText::_('JGLOBAL_FILTER_TYPE_LABEL') . '</span>';
 		$html[] = '		</th>';
-		$html[] = '		<th>';
-		$html[] = '			<span class="acl-action" title="' . JText::_('JGLOBAL_FILTER_TAGS_LABEL') . '">'
-				. JText::_('JGLOBAL_FILTER_TAGS_LABEL') . '</span>';
+		$html[] = '		<th>';
```
Comment (Quy) [inline administrator/components/com_config/model/field/filters.php]: Remove whitespace after `;`

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on fc844fd48f07c769547e2416a9e8dba47833e480<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19104">issues.joomla.org/tracker/joomla-cms/19104</a>.</sub>

**Comment by anuragteapot** [comment]
I have tested this item :white_check_mark: successfully on fc844fd48f07c769547e2416a9e8dba47833e480<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19104">issues.joomla.org/tracker/joomla-cms/19104</a>.</sub>

**Comment by ghost** [comment]
Ready to Commit after two successful tests.

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**